client/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM node:14-alpine
+
+# Create app directory
+WORKDIR /app
+
+# Install app dependencies
+# Copy over package.json
+# Copy over yarn.lock
+COPY package.json ./
+COPY yarn.lock ./
+
+RUN yarn
+
+# Bundle app source
+COPY . .
+
+# Build app and place base api holder
+RUN NEXT_PUBLIC_BASE_API_URL=BASE_API_URL yarn build

```

Comment:
how does this work since the same env variable is already defined in `.env` and not `.env.local`? do actual environment variables automatically take precedence over ones defined in the env file?

Reply:
Yeah environment variables set at system level always take precedence over .env files.

https://create-react-app.dev/docs/adding-custom-environment-variables/#what-other-env-files-can-be-used